include/openssl/pkcs12.h.in

```diff
@@ -45,6 +45,7 @@ extern "C" {
 
 # define PKCS12_MAC_KEY_LENGTH 20
 
+/* The macro expected only used for internal and kept for backwards compatibility */
```

Comment:
`The macro is expected to be used only internally. Kept for backwards compatibility.`

Reply:
Updated,thanks